autokeras/task.py

```diff
@@ -467,3 +467,95 @@ def __init__(self,
             name=name,
             objective=objective,
             seed=seed)
+
+
+class TimeSeriesForecaster(auto_model.AutoModel):
+    """AutoKeras time series data forecast class.
+
+    # Arguments
+        column_names: A list of strings specifying the names of the columns. The
+            length of the list should be equal to the number of columns of the data.
+            Defaults to None. If None, it will be obtained from the header of the csv
+            file or the pandas.DataFrame.
+        column_types: Dict. The keys are the column names. The values should either
+            be 'numerical' or 'categorical', indicating the type of that column.
+            Defaults to None. If not None, the column_names need to be specified.
+            If None, it will be inferred from the data.
+        lookback: Int. The range of history steps to consider for each prediction.
+            For example, if lookback=n, the data in the range of [i - n, i - 1]
+            is used to predict the value of step i. If unspecified, it will be tuned
+            automatically.
+        predict_from: Int. The starting point of the forecast. The number of steps
```

Comment:
The descriptions for these 2 arguments could be improved a bit. In particular, "training data" is confusing as it normally refers to the training dataset (all training samples), but here it refers to the input steps in one sample (that isn't necessarily part of the training data).

E.g.

"Int. The starting point of the forecast for each sample (in number of steps) after the last time step in the input. If N is the last step in the input, then the first step of the predicted output will be `N + predict_from`. Defaults to 1 (which corresponds to starting the forecast immediately after the last step in the input)."